### PR TITLE
Ponomarenko

### PR DESCRIPTION
**Pull request by Sufoku**

### CODE REVIEW

**Review by VlodkoMickovich — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (serhiyivanochko) [inline .gitattributes]: Please remove that file

File: SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
```
Comment (serhiyivanochko) [inline SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs:3]: Empty space

File: SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        IWebElement arrowThree;
+        IWebElement arrowFour;
+        IWebElement arrowFive;
+        IWebElement arrowSix;
+        IWebElement close;
+
+
+
+
+
```
Comment (serhiyivanochko) [inline SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs:49]: Empty spaces

File: SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+            arrowSix = driver.FindElement(By.XPath("/html/body/div[1]/div[2]/div[2]/div[1]/div[3]/div[1]/div/div/div/div[4]/div/div[3]/div[1]/h2/button/span/svg"));
+            checkboxControlFour = driver.FindElement(By.XPath(@"//*[@id='inspector - checkbox - control - 2']"));
+            checkboxControlFive = driver.FindElement(By.XPath(@"//*[@id='inspector - checkbox - control - 3']"));
+
+            block = driver.FindElement(By.ClassName("edit-post-sidebar__panel-tab ")).FindElement(By.LinkText("block"));
+            noBlock = driver.FindElement(By.ClassName("edit-post-sidebar__panel-tab "));
+            close = driver.FindElement(By.XPath("/html/body/div[1]/div[2]/div[2]/div[1]/div[3]/div[1]/div/div/div/div[4]/div/div[2]/button/svg"));
+
+        
+
```
Comment (serhiyivanochko) [inline SSCCSET2019/SSCCSET2019/Pages/AddNewPagesWithSttings.cs:95]: Empty spaces

**Review by serhiyivanochko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**